File: 1-easy/problems/1752-check-if-array-is-sorted-and-rotated/solution.ts
function check(nums: any): boolean | number | string | any {
  """
  :type nums: List[int]
  :rtype: bool
  """
  count = 0
  for (i in Array.from({length: (nums}, (_, i) => i).length)) {
      if (nums[i] > nums[(i+1)%(nums).length]) {
          count += 1
          if (count > 1) {
              return false;
  return true;
}

export { check };

File: 1-easy/problems/1752-check-if-array-is-sorted-and-rotated/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def check(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        count = 0
        for i in range(len(nums)):
            if nums[i] > nums[(i+1)%len(nums)]:
                count += 1
                if count > 1:
                    return False
        return True
